src/montecarlo/load_mcs_RVs.jl

```diff
@@ -0,0 +1,110 @@
+using DataStructures
+
+"""
+This function loads the FUND input parameters stored in datadir into the syntax
+needed for the @defmcs macro. This script writes to a text file which can then 
+be copied into a @defmcs macro.
+"""
+function load_mcs_RVs(; txt_out = joinpath(@__DIR__, "mcs_RVs.txt"), datadir = joinpath(@__DIR__, "../../data"), string_labels = false)
+
+    files = readdir(datadir)
+    filter!(i->i!="desktop.ini", files)
+    parameters = OrderedDict{Any, Any}(splitext(file)[1] => readdlm(joinpath(datadir,file), ',') for file in files)
+
+    open(txt_out, "w") do f 
+        for (k, v) in parameters
+            val = loadparameter(v, string_labels)
+            if val != nothing
+                write(f, "$k = $val\n")
+            end
+        end 
+    end
+
+end
+
```

Comment:
I don't recall if this was intended to be used just once and then maintained as `@defmcs`, or if this will be used repeatedly. If the latter, we should convert this to a macro that emits the `@defmcs` macro, eliminating the writing to text file and the manual copying.